package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "starknetkit",
-  "version": "2.0.0",
+  "version": "1.1.3",
```

Comment:
as @gergold pointed, reverting version 

this will be done automatically with release (will need to add a `BREAKING CHANGE` commit